lib/compilerUtils.js

```diff
@@ -275,9 +275,30 @@ function addCamelSpace(text) {
     return result.substr(0, 1).toUpperCase() + result.substr(1);
 }
 
+const incompatibleMetatags = [
+    ["supportedApps", "unsupportedApps"],
+    ["clientOnly", "serverOnly"],
+];
+
+function validateMetatags(meta, itemType) {
+    if (!meta) {
+        return;
+    }
+    for (const pair of incompatibleMetatags) {
+        if (meta[pair[0]] && meta[pair[1]]) {
+            throw new Error(
```

Comment:
Since this is ultimately being consumed by a Webpack plugin we should also catch the error in `GenerateActivityMetadataPlugin.js` and push it onto the `compilation.errors` array. This provides a slightly more graceful handling than the terminated process and stacktrace you get with an unhandled error.

Reply:
Done